client/packages/common/src/api/GqlContext.tsx

```diff
@@ -55,10 +55,24 @@ const handleResponseError = (errors: ResponseError[]) => {
   throw new Error(details || error?.message || 'Unknown error');
 };
 
+const ignoredQueries = ['refreshToken', 'syncInfo'];
+
+const shouldIgnoreQuery = (definitionNode: DefinitionNode) => {
+  const operationNode = definitionNode as OperationDefinitionNode;
+  if (operationNode.operation !== 'query') return false;
```

Comment:
I was going to ask about variants that don't have operation, but as per latest comment, this check is not needed at all ?

Reply:
not really needed.. being somewhat over cautious.